site/zh/blog/new-in-devtools-106/index.md

```diff
@@ -59,124 +69,178 @@ async increment() {
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/GQ9B11tKBcFc1BxQYW9z.png", alt="自动忽略堆栈跟踪中的已知第三方脚本", width="800", height="425" %}
 
 <!-- Previously, the stack trace included third-party scripts like `zone.js` and `core.mjs`. These are not your source code, they are generated by bundlers (e.g. webpack) or frameworks (e.g. Angular). It took a longer time to identify the root cause of an error.  -->
+相对以前，堆栈跟踪包括第三方脚本，如`zone.js`和`core.mjs`。这些不是您的源代码，它们是由捆绑器（如 webpack）或框架（如 Angular）生成的。需要花费较长的时间来确定错误的根本原因。
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/GQ9B11tKBcFc1BxQYW9z.png", alt="Automatically ignore known third-party scripts in the stack trace", width="800", height="425" %}
+<!-- {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/GQ9B11tKBcFc1BxQYW9z.png", alt="Automatically ignore known third-party scripts in the stack trace", width="800", height="425" %} -->
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/GQ9B11tKBcFc1BxQYW9z.png", alt="自动忽略堆栈跟踪中的已知第三方脚本", width="800", height="425" %}
 
 <!-- Behind the scenes, DevTools ignores third-party scripts based on the new `x_google_ignoreList` property in sourcemaps. Frameworks and bundlers need to supply this information. See [Case Study: Better Angular Debugging with DevTools](/blog/devtools-better-angular-debugging/#x_google_ignorelist-in-angular).  -->
+在幕后，DevTools 根据 sourcemaps 中新的`x_google_ignoreList`属性来忽略第三方脚本。框架和捆绑器需要提供这一信息。参见[案例研究：用 DevTools 进行更好的 Angular 调试]（/blog/devtools-better-angular-debugging/#x_google_ignorelist-in-angular）。
 
 <!-- Optionally, if you prefer to always view full stack traces, you can disable the setting via **Settings** > **Ignore list** > **Automatically add known third-party scripts to ignore list**. -->
+另外，如果您喜欢总是查看完整的堆栈跟踪，您可以通过**设置** > **忽略列表** > **自动添加已知的第三方脚本到忽略列表**来禁用该设置。
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/elkhLqA0KV8pWYFgKk8g.png", alt="Setting to automatically add known third-party scripts to ignore list", width="800", height="516" %}
+<!-- {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/elkhLqA0KV8pWYFgKk8g.png", alt="Setting to automatically add known third-party scripts to ignore list", width="800", height="516" %} -->
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/elkhLqA0KV8pWYFgKk8g.png", alt="设置自动添加已知的第三方脚本忽略列表", width="800", height="516" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/e09e489c2b1233ab424d562abc22f297c6322878 #}
 
-Chromium bug: [1323199](https://crbug.com/1323199)
+<!-- Chromium bug: [1323199](https://crbug.com/1323199) -->
+Chromium 议题: [1323199](https://crbug.com/1323199)
 
 
 <!-- ## Improved call stack during debugging  {: #call-stack } -->
+## 在调试过程中改进调用堆栈 {: #call-stack }
 
 <!-- With the **Automatically add known third-party scripts to ignore list** setting, the call stack now shows only frames that are relevant to your code. -->
+通过**自动添加已知的第三方脚本到忽略列表**设置，现在调用栈只显示与你的代码相关的框架。
```

Comment:
@jecfish Do you think maybe there is a better translation for "frames" here than "框架"? This doesn't block this PR per se because I've seen that DevTools currently translates "frames" to "框架" at other places as well, but I think here it means more closely to "帧", and "堆栈帧", which means exactly call stack frame, is a common technical term that I can find online.